Sources/SwiftDocC/Catalog Processing/GeneratedCurationWriter.swift

```diff
@@ -221,33 +222,38 @@ public struct GeneratedCurationWriter {
                 }
                 return
             }
-            
+
             // Check if the source-language specific task groups need to be emitted individually
             for taskGroup in taskGroups {
                 // Gather all the references for the other task groups
-                let otherReferences = taskGroups.reduce(into: Set(), { accumulator, group in
-                    guard group != taskGroup else { return }
-                    accumulator.formUnion(group.references)
-                })
+                let otherReferences = taskGroups.reduce(
+                    into: Set(),
+                    { accumulator, group in
```

Comment:
Do you know if this is caused by a narrow maximum line width or if there's some other rule involved here (potentially the lack of a training closure made this wrap to multiple lines)?

Regardless, I find this more readable on one line.

Reply:
I think this is a combo of the line length plus the setting that puts a line break between arguments if the line needs to be formatted. The formatting when it's written explicitly using trailing closure syntax though interestingly

Reply:
Unfortunately I couldn't find a setting that makes it use trailing closure syntax wherever possible though.